docs/source/authentication.md

```diff
@@ -58,4 +58,3 @@ that ID:
 1. Via the search bar, find "Azure Active Directory" and open it.
 1. In the overview of that, you will see a field "Tenant ID"
 1. Create an environment variable called `HIML_TENANT_ID`, and set that to the tenant ID you just saw.
-
```

Comment:
Not sure if this is a problem with github displaying, but it looks to me like a newline is actually being removed rather than added here

Reply:
My guess is that there were _two_ newlines at the end. This hook ensures there's one.